build-requirements.txt

```diff
@@ -6,3 +6,4 @@ sh==1.11
 prettytable==0.7.2
 netaddr==0.7.15
 ConcurrentLogHandler
+pyyaml
```

Comment:
No version requirement? I think we should be freezing our reqs (inc the CLH line above)